cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatabaseImpl.java

```diff
@@ -209,7 +209,7 @@
 
     @Override
     public String getDatastoreName() {
-        return this.datastoreDir.getName();
+        return this.datastoreDir.toString();
```

Comment:
Since we are returning the path now, I don't think it is right to have this method called `getName`.  I think `getPath` might be more appropriate.